day18/walk.py

```
@@ -0,0 +1,34 @@
+import random
+from color import turtle_color
+
+class Walk:
+    def __init__(self,turtle):
+        self.turtle = turtle
+        self.turtle.color("black")
+        self.turtle.pensize(5)
+        self.turtle.shape("turtle")
+
+    def walk(self, length, amount):
+        for i in range(length):
```

Ruff (0.13.3)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)